admin-dev/themes/default/template/controllers/translations/helpers/view/translation_errors.tpl

```diff
@@ -69,9 +69,9 @@
 					{l s='You MUST use this syntax in your translations. Here are a few examples:'}
 				</p>
 				<ul>
-					<li>"There are <strong>%d</strong> products": {l s='"%s" will be replaced by a number.' sprintf='%d'}</li>
-					<li>"List of pages in <strong>%s</strong>": {l s='"%s" will be replaced by a string.' sprintf='%s'}</li>
-					<li>"Feature: <strong>%1$s</strong> (<strong>%2$d</strong> values)": {l s='The numbers enable you to reorder the variables when necessary.'}</li>
+					<li>"{l s='There are'} <strong>%d</strong> {l s='products'}": {l s='"%s" will be replaced by a number.' sprintf='%d'}</li>
```

Comment:
Hi, 
Thank you for your PR, maybe you have not noticed that you can use {l s='There are [1]%d[/1] products' tags=['<strong>']}

Best regards.